docs/advanced.rst

```diff
@@ -225,7 +225,9 @@ Transports
 
 *eProsima Fast RTPS* implements an architecture of pluggable transports.
 Current version implements five transports: UDPv4, UDPv6, TCPv4, TCPv6 and SHM.
-By default, when a :class:`Participant` is created, one built-in UDPv4 transport is configured.
+By default, when a :class:`Participant` is created, two built-in transports are
+configured SHM and UDPv4. SHM transport will be used for all communications between
+participants in the same machine, and UDPv4 for external communications.
```

Comment:
```suggestion
configured: SHM and UDPv4. SHM transport is be used for all communications between
participants in the same machine, while UDPv4 is used for external communications.
```

Reply:
"is be used?" I guess you mean "is used"